src/components/Header.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import IconButton from './IconButton';
+
+const Header = () => (
+  <div className="Header">
+    <div className="logo">
+      <h1>Bookstore CMS</h1>
+    </div>
+    <nav className="nav">
+      <ul className="nav-links">
+        <li className="books">BOOKS</li>
+        <li className="catagories">CATEGORIES</li>
```

Comment:
- [ ] Categories
* should display only a button with text "Check status".
* After clicking on the `CATEGORIES` it should display "Check status".